custom/icds_reports/migrations/sql_templates/database_views/disha_indicators.sql

```diff
@@ -0,0 +1,110 @@
+DROP VIEW IF EXISTS disha_indicators CASCADE;
```

Comment:
This will need a migration created to be added to the database

Reply:
Will add